covsirphy/cleaning/pcr_data.py

```diff
@@ -571,6 +572,9 @@ def positive_rate(self, country, province=None, window=7, show_figure=True, file
         except PCRIncorrectPreconditionError:
             raise PCRIncorrectPreconditionError(
                 country=country, province=province, message="Too many missing Tests records") from None
+        # Limit tests records to last date
+        if last_date is not None:
+            subset_df = subset_df.loc[subset_df[self.DATE] <= last_date]
```

Comment:
Thank you for your issues and pull requests!
Please update the test of tests/test_cleaning/test_pcr.py::TestPCRData::test_positive_rate to check `last_date` argument using `@pytest.mark.parametrize`.

In line 577, I think we should replace `last_date` with `pd.to_datetime(last_date)`.

Reply:
For some reason it works too with just `<= last_date`. I will update it though.

Reply:
Oops, I tried it and confirmed it works. Sorry for that.

Reply:
It is better practice to show purpose of usage, I kept `pd.to_datetime()`, thank you.